app/tokens.js

```diff
@@ -8,6 +9,27 @@ export function AuthToken() {
     console.error("File not found, initializing JSON.");
     this.file = {"data":[]};
   }
+  if (convertTokensToUint8Array(this.file.data)) {
+    fs.writeFileSync(FILE_NAME, this.file, "cbor");
+  }
+}
+
+// Convert any existing string/base32-encoded tokens to a Uint8Array, and wrap raw ArrayBuffer tokens
+function convertTokensToUint8Array(data) {
```

Comment:
Given that this isn't a utility function and you're only operating on the `this.file.data` object I'd prefer that this function be integrated into the AuthToken class.